src/CounterInterceptor/README.md

```diff
@@ -0,0 +1,28 @@
+# dotnet-counter-interceptor
+The sample demonstrates: 
+- the use of a Worker Workflow Interceptor that counts the number of Workflow Executions, Child Workflow Executions, and Activity Executions and the number of Signals and Queries. It is based
+off of the [Java sample](https://github.com/temporalio/samples-java/tree/main) located [here](https://github.com/temporalio/samples-java/tree/main/core/src/main/java/io/temporal/samples/countinterceptor)
+- the use of a Client Workflow Interceptor that counts the number of Workflow Executions and the number of Signals and Queries.
+
+## Start local Temporal Server
+```bash
+# run only once
+temporal server start-dev
+```
```

Comment:
```suggestion
To run, first see [README.md](https://github.com/temporalio/samples-dotnet/blob/main/README.md) for prerequisites
```

We like to keep the starting of the local server in one place since it can change and there are multiple options. This can be seen in other samples. (in fact, we will make these run easy on cloud too soon, so we don't want to have to come back and update these)

Reply:
Bump, this was never addressed

Reply:
Bump again, this is still not addressed

Reply:
Finally addressed :)